### ISSUE TITLE
Execution failed for task ':app:transformClassesWithMultidexlistForDebug'

### ISSUE THREAD

**Issue by rashmimhatre100** [react-community/react-native-maps#2328]
Installed react-native-maps using 'npm install react-native-maps --save' but by mistakenly ran 'react-native link'. Got errors solved it by removing user data, cleaning android build and everything went back to normal. But now I need to add new library for which I need to run link command which results in 

errror:Execution failed for task ':app:transformClassesWithMultidexlistForDebug'. 
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':app:transformClassesWithMultidexlistForDebug'.
> com.android.build.api.transform.TransformException: Error while generating the main dex list.

What to do? Please help me out with this. Thanks in advance.

**Comment by arghya95** [comment]
You should have try this for your new library.
`react-native link <LibraryName> <Platform>`

**Comment by rashmimhatre100** [comment]
Okay. Thanks. I am closing this issue as my problem is solved.

**Comment by benjaminreid** [comment]
@rashmimhatre100 How did you solve the issue my I ask? I’ve just run into this too.

**Comment by rashmimhatre100** [comment]
Things I did

1) Added react-native-map
2) By mistakenly ran react-native link
3) Got errors while running react-native run-android, solved it using instructions provided in documentation of package.
4) Then added react-native-image-metedata again tried react-native link and errors started so I tried to unlink and uninstall react-native-image-metedata. But somehow every time when I tried react-native link  react-native-image-metedata automatically was getting added into settings.gradle and many other settings files. And same error mention above i.e app:transformClassesWithMultidexlistForDebug. So came to know that react-native-image-metedata is present in package.json so I removed  react-native-image-metedata  from package.json. And error gone.
